test/simplegraphs/generators/randgraphs.jl

```diff
@@ -364,4 +364,24 @@
         @test isvalid_simplegraph(rog3)
         @test !is_cyclic(rog3)
     end
+
+    @testset "randbn" begin
+        for i in 1:10
```

Comment:
```suggestion
        for i in 0:10
```
while we are it, we could also test that special case.